index.html

```diff
@@ -270,16 +270,6 @@ <h2>Modifications to existing procedures</h2>
         line, as specified in [[RFC5285]], section 5, with direction taken from "e"'s direction
         attribute."
       </p>
-      <p>
-        In the algorithm for generating subsequent offers in [[RTCWEB-JSEP]] section 5.2.2, replace "The
-        RTP header extensions MUST only include those that are present in the most recent answer"
-        with "For each RTP header extension listed in [[\HeaderExtensionsToOffer]] that is
-        also present in the most recent answer, and where direction is not "stopped", generate
```

Comment:
I think you just need to take out "also present in the most recent answer".

Reply:
Done.